build/eslint/xfail.list

```diff
@@ -178,10 +161,6 @@ apps/settings/js/panels/languages/languages.js # line 21, col 10, Error - Unsafe
 apps/settings/js/panels/simcard_manager/simcard_manager.js # line 296, col 6, Error - Unsafe assignment to innerHTML (no-unsafe-innerhtml/no-unsafe-innerhtml)
 apps/settings/js/panels/simpin/simpin.js # line 81, col 6, Error - Unsafe assignment to innerHTML (no-unsafe-innerhtml/no-unsafe-innerhtml)
 apps/sharedtest/test/unit/accessibility_helper_test.js # line 13, col 4, Error - Unsafe assignment to innerHTML (no-unsafe-innerhtml/no-unsafe-innerhtml)
-apps/sharedtest/test/unit/dialer/font_size_manager_test.js # line 106, col 10, Error - Unsafe assignment to innerHTML (no-unsafe-innerhtml/no-unsafe-innerhtml)
```

Comment:
This file is removed, but it looks like it's failing in the linter? Is there updates here?